.jenkins/run_tests.pbs

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+#PBS -N test_cc_recipes
+#PBS -P v45
+#PBS -l storage=gdata/ik11+gdata/cj50+gdata/hh5+gdata/qv56
+#PBS -l walltime=1:00:00
+#PBS -l mem=48GB
+#PBS -l ncpus=4
+#PBS -q normalbw
+#PBS -l wd
+#PBS -W block=true
+#PBS -j oe
+#PBS -o log
+
+export set PYTHONNOUSERSITE=1
+
+module purge
+module use /g/data/hh5/public/modules/
+module load conda/analysis3-unstable
```

Comment:
should we use the stable for the testing?

Reply:
Yes, definitely.

Reply:
Fixed.